Lib/test/test_os.py

```diff
@@ -56,6 +57,11 @@
 except ImportError:
     INT_MAX = PY_SSIZE_T_MAX = sys.maxsize
 
+try:
+    import select
+except ImportError:
+    select = None
```

Comment:
It seems all other modules / tests don't guard against `select` not being available. It looks like all platforms should support it (at least `select.select`).